package.json

```diff
@@ -0,0 +1,69 @@
+{
+  "name": "hof-frontend-assets",
+  "version": "0.1.0",
+  "main": "./bin/index.js",
+  "description": "Frontend JS, CSS and images for a form service",
+  "license": "GPLv3",
+  "author": "UKHomeOfficeForms",
+  "repository": "https://github.com/UKHomeOfficeForms/hof-frontend-assets.git",
+  "bugs": "https://github.com/UKHomeOfficeForms/hof-frontend-assets/issues",
+  "bin": {
+    "hof-frontend-assets": "./bin/index.js"
+  },
+  "scripts": {
+    "test": "npm-run-all lint style browser:test unit:coverage",
+    "browser:test": "karma start --single-run",
+    "unit:test": "mocha test/unit",
+    "cover": "istanbul cover _mocha -- test/unit",
+    "check-coverage": "istanbul check-coverage",
+    "unit:coverage": "npm-run-all cover check-coverage",
+    "snyk": "snyk test",
+    "snyk:dev": "snyk test --dev",
+    "lint": "eslint .",
```

Comment:
That's a really bad solution for something which is trivial to fix by adding the thing we're depending on as a dependency. What possible arguments are there *against* adding eslint as a dependency?

In particular because this is open source software and we're putting arbitrary and undocumented obstacles in place of potential contributors.